src/nuget.config

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear />
     <add key="NuGet official package source" value="https://api.nuget.org/v3/index.json" />
-    <add key="platform.uno dev" value="https://uno-platform.pkgs.visualstudio.com/Uno%20Platform/_packaging/unoplatformdev/nuget/v3/index.json" />
```

Comment:
It looks like `platform.uno dev` feed isn't necessary. Once it's deleted, all nuget configs become the same (official nuget feed + PackageCache). So, I reduced them to a single file.